.github/CODEOWNERS

```diff
@@ -40,8 +40,6 @@
 /packages/rewriter/  @cypress-io/end-to-end
 /packages/root/  @cypress-io/end-to-end
 /packages/runner/  @cypress-io/end-to-end
-/packages/runner-ct/  @cypress-io/component-testing
-/packages/runner-shared/  @cypress-io/end-to-end
 /packages/server/  @cypress-io/end-to-end
 /packages/socket/  @cypress-io/end-to-end
 /packages/static/  @cypress-io/end-to-end
```

Comment:
FYI some context - we ideally should delete ALL of runner/runner-shared, and bundle the entire app with the Vite stack.

The reason we are stuck like this is explained a little here: https://github.com/cypress-io/cypress/pull/20663#issuecomment-1073532399

Basically some of the code in driver and possibly reporter are not really ESM compatible - webpack does something different to Vite to resolve the modules, which is why it works in webpack, but we cannot import to Vite.

Not sure if this is something you are looking to investigate more @sainthkh but if you do please let me know, I'd like to follow along/help out - I'd really like to get away from this "inject webpack bundle" mess at some point, but it's quite hard. Likely making driver 100% ESM (including dependencies) would be a good first step.

Reply:
It seems that we have 4 things to do to delete the `runner` pacakge.

- [ ] Migrate `dom`. -> I think `driver/src/dom` might be a good place or move it to `app` directly.
- [ ] Reimplement Studio -> #22870 
- [ ] ESM-ify `driver` -> I agree it's the biggest problem. 
- [ ] Let `reporter` compile its own styles. -> Move `runner/webpack.config.ts` to `reporter`.

Reply:
Dom -> are you going do to this here?
Studio -> I am looking into this now
ESMify driver -> agree, hard
Reporter styles -> do this separately I think?

Reply:
I read the `dom` code and it seems that it's better to be a separate PR.

It currently renders highlight by

1. Creating the container node with jQuery.
2. Rendering the Highlight DOM with React.

I can simply migrate `dom` to `app` and use `unified-runner`. But I don't think it's not our goal. It's just hiding and postponing. 

To get migrating `dom` done, I need to do these things:

- [ ] Remove JQuery from `dom` and `dimensions`.
- [ ] Move the current JQuery version of `dimensions` to `driver/src/dom` because it's used in `driver/src/dom/blackout`.
- [ ] Migrate `Highlight` from `react` to `vue` with our `Tooltip` component. 
- [ ] Migrate `dom` functions to `app/runner/aut-iframe`. Leave `studio`-related functions.
- [ ] Clean up unified-runner `dom` and `CypressJQuery`. They're only used inside `aut-iframe`. I guess they can be removed safely from `unified-runner`.

I think it's a big change and outside the scope of this PR. 

----

As for reporter styles, it's safe to do it after ESM-ifying `driver`. We're trying to remove `webpack`. I don't think we don't need to create one more `webpack.config.ts` now.

Reply:
I opened the new PR #23187 to migrate `dom.js`. It's currently empty because I'm now reading and planning.